### ISSUE TITLE
Open Project Backend - Week 2

### ISSUE THREAD

**Issue by michelaburns** [codeu-spring2019/whats-that-doggo#4]
- [x] Create an AutoML dataset and upload the CSV.

- [x] Train, and then test the model with a random image.

**Comment by michelaburns** [comment]
As stated in the group chat, I've been going through the "Cloud Codelab" for Auto ML Vision [https://codelabs.developers.google.com/codelabs/cloud-automl-vision-intro/index.html?index=..%2F..index#0 ](url) . So, I believe I will be able to finish at least that first bullet point with the help of that lab. I'll keep you guys updated!

**Comment by elithrar** [comment]
Thanks for the status update + for creating these tasks!

**Comment by michelaburns** [comment]
Hi guys! Just to make sure this issue is properly documented, I just wanted to add a brief synopsis of an issue I faced while working on checkpoint #1. This is more thoroughly documented in the Google Chat and within the document. AutoML has guidelines for the buckets you can import images from. So I ended up having to transfer the data from our Whats-That-Doggo bucket to our vcm bucket. I used this    [https://cloud.google.com/storage/docs/moving-buckets](url)  to complete that

**Comment by michelaburns** [comment]
Finished creating an AutoML dataset for the most part (using `dogs_dataset_official.csv`), however, only 20491 images uploaded, when the csv had 20580 images. So, there are 89 images "missing". 

There two reasons for the missing images
- AutoML has a warning that states, "Warning: Importing images". When clicked on, AutoML tells us that it detected 69 duplicates. AutoML automatically deletes duplicates, so if there are 69 instances of duplicates, that means that 69 images would be kept and 69 duplicates would be deleted. So, this leaves us with 20 more unaccounted images (89-69).

- Within our bucket, the folder `dogs_dataset/` contains our AutoML dataset. In this folder, there is a log file with 37 errors that state, "Dataset is not multilabel but more than one label assigned to __". After comparing the files from the log to the csv, NONE of them have multiple labels, so I'm not sure why this is occurring. I also checked to see if any of the files in the logs were duplicate images, but they weren't. So I'm not sure why this happened. An online forum recommended remaking the dataset, and this time enabling multi-class classification, so I tried that with `dogs_dataset_2`. However, this dataset came out with the same number of images. However, Warning only listed duplicates (and not label issues)

As a result, I think I will move forward with dogs_dataset_2 since it just has the duplicates issue, whereas the other one had the weird multilabel and duplicate issues.
